src/tests/ftest/aggregation/aggregation_basic.py

```diff
@@ -2,24 +2,7 @@
 """
   (C) Copyright 2020 Intel Corporation.
```

Comment:
Looks like some of these didn't get hit by the copyright git hook?

Reply:
You are right, Kris, actually many, if not all .py files header are not properly updated with the year.
Currently the daos/.git/hooks/pre-commit which links to utils/githooks/pre-commit is not able to
handle python files yet, will have to wait for future update when this ticket is solved:
https://jira.hpdd.intel.com/browse/DAOS-5705

Reply:
Okay, good to know there's already a ticket.